src/components/json-tree.js

```diff
@@ -50,12 +50,22 @@ export default class JsonTree extends LitElement {
       .tree > .open-bracket {
         margin-left: -2px;
       }
+      .open-bracket:not(.collapsed):hover > .key > .open-bracket, 
+      .open-bracket:not(.collapsed):hover ~ .inside-bracket-wrapper > .close-bracket,
       .open-bracket:hover {
         color:var(--primary-color);
         background-color:var(--hover-color);
         border: 1px solid var(--border-color);
       }
-
+      .open-bracket:not(.collapsed):hover ~ .inside-bracket-wrapper > .inside-bracket {
+        border-left: 1px solid var(--fg3);
+      }
+      .close-bracket {
+        display:inline-block;
+        border: 1px solid transparent;
+        border-radius:3px;
+        font-family: var(--font-mono);
```

Comment:
Can we also explicitly set the font-family on the open-bracket. I know it is included in the `key`, but if feels weird without it, and who knows if we end up pulling that object into its own tag, it could break the styling at that point if we forget.